changelog.txt

```diff
@@ -10,6 +10,7 @@
 * Fix - Prevent Payment Request buttons from showing up in Composite Product pages.
 * Update - Updated @woocommerce/experimental package to v2.1.0.
 * Add - Onboarding flows on the admin WooCommerce > Subscriptions screen for stores with no subscriptions yet.
+* Add - IPP Styled Receipts: Merchant brand settings (API)
```

Comment:
This can perhaps be reworded as "Update - Get and update merchant branding for in-person payment receipts via the Settings API endpoint."